nginx/nginx.go

```diff
@@ -86,13 +86,14 @@ func (n *nginxUpdater) signalIfRequired() {
 // Nginx implementation
 type nginxUpdater struct {
 	Conf
-	running              util.SafeBool
-	lastErr              util.SafeError
-	metricsUnhealthy     util.SafeBool
-	initialUpdateApplied initialUpdateApplied
-	doneCh               chan struct{}
-	nginx                *nginx
-	updateRequired       util.SafeBool
+	running              	util.SafeBool
+	lastErr              	util.SafeError
+	metricsUnhealthy     	util.SafeBool
+	initialUpdateApplied 	initialUpdateApplied
+	initialUpdateAttempted	util.SafeBool
+	doneCh               	chan struct{}
+	nginx                	*nginx
+	updateRequired       	util.SafeBool
 }
 
 type initialUpdateApplied struct {
```

Comment:
I think we could rename this if we have 'initialUpdateAttempted' now. We can call it something like `nginxStarted`.